pyzbar/pyzbar.py

```diff
@@ -144,9 +145,22 @@ def decode(image, symbols=None):
                 while symbol:
                     data = string_at(zbar_symbol_get_data(symbol))
                     symbol_type = ZBarSymbol(symbol.contents.value).name
+
+                    location = []
+                    loc = zbar_symbol_get_loc_size(symbol)
+                    if loc:
```

Comment:
I am curious - does `zbar_symbol_get_loc_size` ever return `0`? I have done some limited testing on Mac OS X and have always seen a non-zero return.